### DISCUSSION TITLE
Homing Speed

### DISCUSSION THREAD

**Discussion by YankeeDeMierda** [Jyers/Marlin#1235] [Q&A] [answered]
What speed and acceleration does the machine home at? The firmware homing is very reliable but using prusaslicer on an ender 3 v2, the home sequence is often higher than it should be. I think by forcing the same z speed into the "machine limits" on prusaslicer I could fix this.

**Comment by Jyers** [comment, marked as answer]
Prusa shouldn't be overriding your limits, but if you need to find the probing speed it can be found in the configuration templates in the source code. Just do a find for "probing"
